.idea/src/lesson12/Task2/App.java

```diff
@@ -0,0 +1,122 @@
+package lesson12.Task2;
+
+import java.util.*;
+
+public class App {
+
+    public static void main(String[] args) {
+        System.out.println("Enter the name of month from keyboard =  ");
+        Scanner one = new Scanner(System.in);
+        String newMonthFromKeyboard = one.next();
+        Month month = Month.valueOf(newMonthFromKeyboard.toUpperCase());
+
+        List<Month> list = new ArrayList<Month>();
+
+        list.add(0, Month.JANUARY);
+        list.add(1, Month.FEBRUARY);
+        list.add(2, Month.MARCH);
+        list.add(3, Month.APRIL);
+        list.add(4, Month.MAY);
+        list.add(5, Month.JUNE);
+        list.add(6, Month.JULY);
+        list.add(7, Month.AUGUST);
+        list.add(8, Month.SEPTEMBER);
+        list.add(9, Month.OCTOBER);
+        list.add(10, Month.NOVEMBER);
+        list.add(11, Month.DECEMBER);
+
+        if (list.contains(month)) {
+            System.out.println("The name of entered month is : " + month.name());
+        } else {
+            System.out.println("No such month");
+        }
+
+        // Виводимо кількість днів для місяця введеного з консолі
+        System.out.println("There " + " " + month.days + " days in " + month.name());
+
+
+        // Виводимо місяці які мають таку ж кількість днів
```

Comment:
Instead of making comment is it better to have a method with a self-explained name